### PR TITLE
Add contao-themes-shop/vacancy-calendar meta data

### PR DESCRIPTION
**Pull request by contao-themes-shop**

### CODE REVIEW

File: meta/contao-themes-shop/vacancy-calendar/en.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+de:
```
Comment (fritzmg) [inline meta/contao-themes-shop/vacancy-calendar/en.yml:1]:
```suggestion
en:
```

**Review by fritzmg — CHANGES_REQUESTED** [review]

**Comment by contao-themes-shop** [comment]
@fritzmg is fixed now :-)

File: meta/contao-themes-shop/vacancy-calendar/en.yml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+en:
+    title: Vacancy calendar
+    description: >
+        Allows the display of vacancies for different objects in a simple and convenient way. For each object a frontend-module is used.
+    keywords:
+        - Vacancy calendar
+        - availibility
```
Comment (xchs) [inline meta/contao-themes-shop/vacancy-calendar/en.yml:7]:
```suggestion
        - availability
```

File: meta/contao-themes-shop/vacancy-calendar/en.yml [hunk 3]
```diff
@@ -0,0 +1,10 @@
+en:
+    title: Vacancy calendar
+    description: >
+        Allows the display of vacancies for different objects in a simple and convenient way. For each object a frontend-module is used.
```
Comment (fritzmg) [inline meta/contao-themes-shop/vacancy-calendar/en.yml:4]:
```suggestion
        Allows the display of vacancies for different objects in a simple and convenient way. For each object a front end module is used.
```
The spell checkers complains about `frontend`. In English, `front end` is the correct term. Hence this change request.

**Review by fritzmg — CHANGES_REQUESTED** [review]

**Review by fritzmg — APPROVED** [review]

**Pull request merged**